client/src/components/Hero.module.css

```diff
@@ -9,8 +9,8 @@
     display: flex;
     flex-wrap: wrap;
     align-items: center;
-    
+    flex-direction: row-reverse;
```

Comment:
This makes the component code harder to read. Components have a certain order in the component file, but are reversed by a class definition which is a bit hidden in a style file.